mne_bids/tests/test_read.py

```diff
@@ -324,42 +326,122 @@ def test_handle_info_reading():
 
 @pytest.mark.filterwarnings(warning_str['nasion_not_found'])
 @pytest.mark.filterwarnings(warning_str['channel_unit_changed'])
-def test_handle_coords_reading():
-    """Test reading coordinates from BIDS files."""
+def test_handle_ieeg_coords_reading():
+    """Test reading iEEG coordinates from BIDS files."""
     bids_root = _TempDir()
 
     data_path = op.join(testing.data_path(), 'EDF')
     raw_fname = op.join(data_path, 'test_reduced.edf')
+    bids_fname = bids_basename + "_ieeg.edf"
 
     raw = mne.io.read_raw_edf(raw_fname)
 
     # ensure we are writing 'ecog'/'ieeg' data
     raw.set_channel_types({ch: 'ecog'
                            for ch in raw.ch_names})
 
+    # coordinate frames in mne-python should all map correctly
     # set a `random` montage
     ch_names = raw.ch_names
     elec_locs = np.random.random((len(ch_names), 3)).astype(float)
     ch_pos = dict(zip(ch_names, elec_locs))
-    montage = mne.channels.make_dig_montage(ch_pos=ch_pos,
-                                            coord_frame="mri")
-    raw.set_montage(montage)
-    write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
+    coordinate_frames = ['mri', 'ras']
+    for coord_frame in coordinate_frames:
+        # XXX: Currently mne-bids can't handle two electrodes files
```

Comment:
there can be two electrode files associated with a single raw file? BIDS allows this? for what purpose?

Reply:
https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/04-intracranial-electroencephalography.html#multiple-coordinate-systems